src/logger.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-undef */
+const fs = require("fs");
+const path = require("path");
+const { performance } = require("perf_hooks");
+
+const date = new Date();
+
+const fileName = `${date.getFullYear()}-${date.getMonth()}-${date.getDay()}-log`;
+
+const pathLogDir = path.join(process.env.HOME, "EventsExpressNode-logs");
+const pathLogFile = path.join(pathLogDir, fileName);
+
+const isAccessible = (path) => {
+  return fs.accessSync(path);
+};
+
+const logger = () => {
+  try {
+    try {
+      isAccessible(pathLogDir);
+    } catch (error) {
+      fs.mkdirSync(pathLogDir);
+    }
+
+    try {
+      isAccessible(pathLogFile);
+    } catch (error) {
+      fs.writeFileSync(pathLogFile, "");
+    }
+
+    return (req, _, next) => {
```

Comment:
Move it outside of try catch block. It will not catch errors properly

Reply:
yes